src/components/ArticleList.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+import { getAllArticles } from "./apiCalls";
+import { ArticleCard } from "./ArticleCard";
+
+export const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
Should really start this as true as the component will be loading on initial render.